lib/repl.js

```diff
@@ -365,6 +367,45 @@ function REPLServer(prompt,
     };
   }
 
+  function filterInternalStackFrames(error, structuredStack) {
+    // search from the bottom of the call stack to
+    // find the first frame with a null function name
+    if (typeof structuredStack !== 'object')
+      return structuredStack;
+    const idx = structuredStack.reverse().findIndex(
+      (frame) => frame.getFunctionName() === null);
+
+    // if found, get rid of it and everything below it
+    structuredStack = structuredStack.splice(idx + 1);
+    return structuredStack;
+  }
+
+  function prepareStackTrace(fn) {
+    return (error, stackFrames) => {
+      const frames = filterInternalStackFrames(error, stackFrames);
+      if (fn) {
+        return fn(error, frames);
+      }
+      frames.push(error);
+      return frames.reverse().join('\n    at ');
+    };
+  }
+
+  /**
+   * Used to parse and execute the Node REPL commands.
+   *
+   * @param {keyword} keyword The command entered to check.
+   * @return {Boolean} If true it means don't continue parsing the command.
+   */
+  REPLServer.prototype.parseREPLKeyword = function(keyword, rest) {
```

Comment:
Isn't this function already defined here?

https://github.com/nodejs/node/blob/edb03cb65d51e336713d6af1134a7394c0776635/lib/repl.js#L1053-L1066

Reply:
Hmm - that's disconcerting enough for me to want to start over on this. I'll take a look